docs/developer-docs/latest/developer-resources/database-apis-reference/entity-service/populate.md

```diff
@@ -10,18 +10,18 @@ The [Entity Service API](/developer-docs/latest/developer-resources/database-api
 
 ## Basic populating
 
-To populate all the root level relations, use `populate: true`:
+To populate specific relation or component, add his name to `populate`:
 
 ```js
-strapi.entityService('api::article.article').findMany({
-  populate: true,
+const entries = await strapi.entityService.findMany('api::article.article', {
+  populate: 'componentA',
```

Comment:
can you change this to `'*'` instead to keep the example similar (the goal is to populate multiple fields without specifying them)